features/networking/sctp.feature

```diff
@@ -0,0 +1,59 @@
+Feature: SCTP related scenarios
+
+  # @author weliang@redhat.com
+  # @case_id OCP-28757
+  @admin
+  @destructive
+  Scenario: Establish pod to pod SCTP connections		
+    Given I store the workers in the :workers clipboard
+    And the Internal IP of node "<%= cb.workers[1].name %>" is stored in the :worker1_ip clipboard
```

Comment:
This step seems redundant

Reply:
@anuragthehatter New version has three new steps about node's information, do you still think they are redundant? 
 Given I store the workers in the :workers clipboard
    And I store the number of worker nodes to the :num_workers clipboard
    And the Internal IP of node "<%= cb.workers[1].name %>" is stored in the :worker1_ip clipboard